azure-pipelines-samples.yml

```diff
@@ -134,7 +134,65 @@ jobs:
         createLogFile: true
       condition: and(succeededOrFailed(), eq(variables['BuildConfiguration'], 'Release'))
 
-    # TODO: Add previously failing build tasks
+    - task: VSBuild@1
+      displayName: 'Build UI test solution Testing/**/SamplesTest.sln'
+      inputs:
+        solution: 'Testing/**/SamplesTest.sln'
```

Comment:
IIRC, this test was supposed to actually run MNIST and draw into the frame - does that still happen somewhere? It looks like this task builds the sample, but does it actually run the samplesTest app somewhere?

Reply:
I think it just builds. This was the same task that was there previously; the only change I made was the addition of ' /t:Restore,Clean,Build' to the end of the msbuildArgs.

Maybe there was a task to actually run it in a different pipeline? You can see what the task looked like before I removed it [here](https://github.com/microsoft/Windows-Machine-Learning/pull/422/files#diff-2d5f7450f4896cae0a954e6d1e85749d2815aa90b4edb7fb9ddf110aadf65d54L71).